pennylane/_device.py

```diff
@@ -359,9 +369,27 @@ def var(self, observable, wires, par):
             par (tuple): parameters for the observable
 
         Returns:
-            float: expectation value
+            float: variance
         """
-        raise NotImplementedError
+        raise NotImplementedError("Returning variances from QNodes not currently supported by {}".format(self.short_name))
+
+    def sample(self, observable, wires, par, n=None):
+        """Return a sample of an observable.
+
+        For plugin developers: this function should return the result of an evaluation
+        of the given observable on the device.
```

Comment:
I have a feeling that after merging master, you might have a merge conflict here. Since this docstring will also appear in inherited classes (i.e., plugins), we have removed this sentence from `.var` and `.expval`, so you will need to make the equivalent change to `.sample` to bring it in-line.

Reply:
Good to know!